include/psa/crypto_compat.h

```diff
@@ -105,11 +105,11 @@ static inline int psa_key_handle_is_null(psa_key_handle_t handle)
  *         permission to access it. Note that this specification does not
  *         define any way to create such a key, but it may be possible
  *         through implementation-specific means.
- * \retval #PSA_ERROR_COMMUNICATION_FAILURE
- * \retval #PSA_ERROR_CORRUPTION_DETECTED
- * \retval #PSA_ERROR_STORAGE_FAILURE
- * \retval #PSA_ERROR_DATA_INVALID
- * \retval #PSA_ERROR_DATA_CORRUPT
+ * \retval #PSA_ERROR_COMMUNICATION_FAILURE \emptydescription
```

Comment:
Is `\emptydescription` some special Doxygen command? If so, what versions of Doxygen support it?

Reply:
It's a custom command defined in the first commit.

Reply:
Ah - there are a lot of comments there, but they didn't tell me the one thing I needed to know as a non-expert in Doxygen!